care/users/api/viewsets/users.py

```diff
@@ -371,3 +384,20 @@ def check_availability(self, request, username):
         if User.check_username_exists(username):
             return Response(status=status.HTTP_409_CONFLICT)
         return Response(status=status.HTTP_200_OK)
+
+    @extend_schema(tags=["users"])
+    @action(detail=True, methods=["POST"], permission_classes=[IsAuthenticated])
+    def profile_picture(self, request, username):
+        user = self.get_object()
+        serializer = UserImageUploadSerializer(user, data=request.data)
+        serializer.is_valid(raise_exception=True)
+        serializer.save()
+        return Response(status=status.HTTP_200_OK)
+
+    @extend_schema(tags=["users"])
+    @profile_picture.mapping.delete
+    def profile_picture_delete(self, *args, **kwargs):
+        user = self.get_object()
+        user.profile_picture_url = None
+        user.save()
+        return Response(status=status.HTTP_204_NO_CONTENT)
```

Comment:
Authorization missing

Reply:
not able to figure out what the issue with my comment code , can you have look into it

Reply:
not able to figure out what the issue with commented code , can you please have a look into it

Reply:
updated